ietf/review/utils.py

```diff
@@ -439,9 +439,9 @@ def assign_review_request_to_reviewer(request, review_req, reviewer, add_skip=Fa
 
     email_review_request_change(
         request, review_req,
-        "%s %s assignment: %s" % (review_req.team.acronym.capitalize(), review_req.type.name,review_req.doc.name),
+        "For %s, %s %s review by %s assigned: %s" % (reviewer.person.ascii, review_req.team.acronym.capitalize(), review_req.type.name, review_req.deadline, review_req.doc.name),
```

Comment:
Why ASCII?

Reply:
Good point, could be reviewer.person.name and have my nice "Éric" ;-)